src/Service/BookingServiceInterface.php

```diff
@@ -118,4 +119,11 @@ public function getUserBookingFromApiData(array $data): UserBooking;
      * @return string
      */
     public function createBodyUserId(string $id): string;
+
+    /**
+     * Get all future bookings.
+     *
+     * @return mixed
+     */
+    public function getAllFutureBookings(string $token): GraphResponse;
```

Comment:
This function should return a non-graph response, e.g. an array of UserBookings.
The conversion from graphresponse to array should be handled in `src/Service/MicrosoftGraphBookingService.php`

Reply:
Fixed